native-stack/README.md

```diff
@@ -256,6 +256,102 @@ Sets the current screen's available orientations and forces rotation if current
 
 Defaults to `default`.
 
+### Search bar (iOS only)
+
+The search bar is just a `searchBar` property that can be specified in the navigator's `screenOptions` or an individual screen's `options`. Search bars are rarely static so normally it is controlled by passing an object to `searchBar` navigation option in the component's body.
+
+Search bar is only supported on iOS.
+
+Example: 
+
+```js
+React.useEffect(
+  navigation.setOptions({
+    searchBar: {
+      // search bar options
+    }
+  });
+}, [navigation]);
+```
+
+Supported properties are described below.
+
+#### `autoCapitalize`
+
+Controlles whether text is automatically auto-capitalized as it is entered by the user.
```

Comment:
```suggestion
Controls whether text is automatically auto-capitalized as it is entered by the user.
```

Reply:
Resolved in https://github.com/software-mansion/react-native-screens/pull/758/commits/49247f00af70303e425be0695dba328f3e2b4d4a